src/main/java/teammates/common/util/SanitizationHelper.java

```diff
@@ -226,6 +226,21 @@ public static String desanitizeFromHtml(String sanitizedString) {
     }
 
     /**
+     * Returns true if string contains no characters to be sanitized and contains substrings of sanitized characters
+     */
+    public static boolean isSanitizedHtml(String string) {
```

Comment:
It looks like this function is used only on the script. Can you move it there instead? It's not really preferable to put methods only used in non-production to the production code base.
(yes that means you don't need to test it)